include/elements/2d/quadrilateral_lme_element.h

```diff
@@ -190,9 +190,10 @@ class QuadrilateralLMEElement : public QuadrilateralElement<2, 4> {
   //! Assign nodal connectivity property for LME elements
   //! \param[in] beta Coldness function of the system in the range of [0,inf)
   //! \param[in] radius Support radius of the kernel
+  //! \param[in] anisotropy Shape function anisotropy (F^{-T}F^{-1})
   //! \param[in] nodal_coordinates Coordinates of nodes forming the cell
   void initialise_lme_connectivity_properties(
-      double beta, double radius,
+      double beta, double radius, const unsigned anisotropy,
```

Comment:
Make it as the last argument without const and with default to false. So:
`bool anisotropy = false`